composer.json

```diff
@@ -66,9 +66,15 @@
         "illuminate/cache": "^6.0",
         "squizlabs/php_codesniffer": "3.*",
         "php-parallel-lint/php-parallel-lint": "^1.0",
-        "meyfa/phpunit-assert-gd": "^2.0.0",
-        "dms/phpunit-arraysubset-asserts": "^0.1.0"
+        "meyfa/phpunit-assert-gd": "^2.0.0|dev-master",
+        "dms/phpunit-arraysubset-asserts": "^0.1.0|^0.2.1"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/bennothommo/phpunit-assert-gd"
```

Comment:
If we have to retain this into release I would prefer it be under octoberrain

Reply:
Sure thing. Hopefully it gets merged though.